engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -780,16 +780,16 @@ public void monitorHeartbeat(boolean waitForRecovery)
           //LOG.debug("{} {} {}", c.getExternalId(), currentTms - sca.createdMillis, this.vars.heartbeatTimeoutMillis);
           // container allocated but process was either not launched or is not able to phone home
           if (currentTms - sca.createdMillis > 2 * this.vars.heartbeatTimeoutMillis) {
-            LOG.error("Container {}@{} startup timeout ({} ms).", c.getExternalId(), c.host, currentTms - sca.createdMillis);
+            LOG.info("Container {}@{} startup timeout ({} ms).", c.getExternalId(), c.host, currentTms - sca.createdMillis);
```

Comment:
@tweise I disagree that it is info level message. With the normal flow of operations, heartbeat should not timeout. It is an indication of slow/busy Yarn cluster or improper (for that cluster) timeout settings. If not error, it should be a warning.